markets/legacy-market/contracts/LegacyMarket.sol

```diff
@@ -222,25 +224,7 @@ contract LegacyMarket is ILegacyMarket, Ownable, UUPSImplementation, IMarket, IE
         uint256 cratio = (collateralMigrated * v3System.getCollateralPrice(address(oldSynthetix))) /
             debtValueMigrated;
 
-        // if the account needs to be liquidated, liquidate it here by unlocking the debt to all accounts and moving to a
         if (
-            cratio < v3System.getCollateralConfiguration(address(oldSynthetix)).liquidationRatioD18
```

Comment:
If we delete this line how does the liquidation workflow work? I mean how can one force a v2x liquidation, without hitting reverts due to not owning the account when calling migrate?